moveit_experimental/moveit_jog_arm/README.md

```diff
@@ -51,8 +51,10 @@ twist:
 
 If you see a warning about "close to singularity", try changing the direction of motion.
 
+The `change_control_dimensions` service allows you to select which directions (in the EE output frame) to allow jogging motion in. The input is an array of 6 booleans ordered as [translation-x, translation-y, translation-z, rotation-x, rotation-y, rotation-z]. Setting one of the dimensions to `false` will set the velocity to 0 in that direction, regardless of user input.
```

Comment:
Rather than maintaining this README and the tutorials, can we move all this documentation into the tutorial?

https://ros-planning.github.io/moveit_tutorials/doc/arm_jogging/arm_jogging_tutorial.html

Reply:
Added this information to that tutorial, [here](https://github.com/ros-planning/moveit_tutorials/pull/433).